crates/virtual-dom-rs/tests/diff_patch.rs

```diff
@@ -170,14 +170,26 @@ fn replace_element_with_text_node() {
     .test();
 }
 
-//// https://github.com/chinedufn/percy/issues/68
-//#[wasm_bindgen_test]
-//fn text_root_node() {
-//    DiffPatchTest {
-//        desc: "Patching of text root node works",
-//        old: html! { Old text },
-//        new: html! { New text },
-//        override_expected: None,
-//    }
-//        .test();
-//}
+// https://github.com/chinedufn/percy/issues/68
+#[wasm_bindgen_test]
+fn text_root_node() {
+    DiffPatchTest {
+        desc: "Patching of text root node works",
+        old: html! { Old text },
+        new: html! { New text },
+        override_expected: None,
+    }
+        .test();
+}
+
+// https://github.com/chinedufn/percy/issues/62
```

Comment:
I think that I added a test for this already in this file - unless I'm missing something here https://github.com/chinedufn/percy/blob/7934e8b10cc91b109bf732ba2a5d463d939f1510/crates/virtual-dom-rs/tests/diff_patch.rs#L161-L171

---

Also right now this doesn't prepend.

It will replace the `<br>` with `text` and then append a new `<br>`

In the future we might support something like [React keys](https://reactjs.org/docs/lists-and-keys.html) though 

Not relevant here... just an FYI

---

Reply:
I see. I'll remove the test, I actually didn't mean to commit it!